### PR TITLE
fix handling of whitespace-only SeqRecord descriptions

### PR DESCRIPTION
**Pull request by aowi**
This pull request addresses issue #1624 

I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

I have read the ``CONTRIBUTING.rst`` file and understand that AppVeyor and
TravisCI will be used to confirm the Biopython unit tests and ``flake8`` style
checks pass with these changes.

I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

**Comment by peterjc** [comment]
There seems to be an unrelated stockholm fix on this pull request for SeqRecord descriptions?

**Comment by aowi** [comment]
It isn't unrelated. Adding the test-case with a whitespace description broke the Stockholm test as mentioned since it also doesn't handle this case. This was my attempt at doing something sensible, but I'm not sure at all if it's correct.

Alternatively, I could just make the test accept the exception generated by the stockholm writer. The underlying bug would just remain for somebody else to stumble upon.

**Comment by peterjc** [comment]
Ah. OK. thanks for the clarification - that does make sense now, SeqIO and AlignIO are somewhat inter-linked.

File: Bio/AlignIO/StockholmIO.py [hunk 1]
```diff
@@ -454,7 +454,16 @@ def __next__(self):
                 elif line[:5] == '#=GS ':
                     # Generic per-Sequence annotation, free text
                     # Format: "#=GS <seqname> <feature> <free text>"
-                    seq_id, feature, text = line[5:].strip().split(None, 2)
```
Comment (peterjc) [inline Bio/AlignIO/StockholmIO.py]: Taking the specification literally, the free text could be spaces? If so, does this cause problems elsewhere?

```
seq_id, feature, text = line[5:].split(None, 2)
```

Also, should we not make a similar change to the ``"#=GF `` case above?

**Comment by peterjc** [comment]
Apologies that this fell off my to-do list, and there are now multiple conflicts so a rebase wouldn't be trivial. It might be worthwhile revisiting, but I'm closing the stale pull request.
